### PR TITLE
[SDPA-2995] Preview Links - Test Environment. DO NOT MERGE.

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-2995

### Changed

1.  This PR is https://github.com/dpc-sdp/ripple/pull/793 but with the .env set to https://nginx-php-content-vic-pr-955.lagoon.vicsdp.amazee.io/

### CODE REVIEW

File: packages/ripple-nuxt-tide/modules/preview/module.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          maxAge: 300,
+          name: 'X-OAuth2-Authorization'
+        },
+        refreshToken: {
+          property: 'refresh_token',
+          maxAge: 1209600
+        },
+        responseType: 'code',
+        grantType: 'authorization_code',
+        clientId: 'dc881486-c14a-4b92-a0d0-e5dcd706f5ad',
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/modules/preview/module.js:45]: Hey @alan-cole Can we replace this by env var `CONTENT_API_OAUTH_CLIENT_ID`?

Comment (anthony-malkoun) [reply packages/ripple-nuxt-tide/modules/preview/module.js:45]: @tim-yao upon reviewing the use of this client id, I'm actually going to go with having the one value all SDP sites. Looking at the reference at https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/ the client id is public but _shouldn't_ be obvious or guessable. If Google, Instagram etc. can have a static client id then I think SDP can too :)

Comment (tim-yao) [reply packages/ripple-nuxt-tide/modules/preview/module.js:45]: @anthony-malkoun thanks. Sounds good to me!

**Comment by alan-cole** [comment]
Closing as Preview links is now part of develop.

**Pull request closed without merging**